docs/sources/alerting/alerting-rules/create-data-source-managed-rule.md

```diff
@@ -165,7 +165,3 @@ All alert rules and instances, irrespective of their labels, match the default n
    Add custom labels by selecting existing key-value pairs from the drop down, or add new labels by entering the new key or value.
 
 {{< docs/shared lookup="alerts/configure-notification-message.md" source="grafana" version="<GRAFANA_VERSION>" >}}
-
```

Comment:
I moved this to the `docs/sources/alerting/monitor-status/view-alert-rules.md` as these actions belong to the list view page.